doc/lua_api.txt

```diff
@@ -7312,6 +7316,9 @@ Used by `Player:hud_add`. Returned by `Player:hud_get`.
 
         size = { x=100, y=100 },
         -- Size of element in pixels
+
+        z_index = 0,
+        -- Z index : lower z-index HUDs are displayed under higher z-index HUDs
```

Comment:
`under` is confusing. I'd personally prefer using the words "behind" or "in front of", depending on the behaviour of this property.

Reply:
Right ! Fixed :)

Reply:
The docs are now much easier to understand, thanks!